testutil/db.go

```diff
@@ -37,10 +37,21 @@ const (
 	envKeyAwsRegion       = "AWS_DEFAULT_REGION"
 )
 
-var keepDB = flag.Bool("keep-db", false, "keep test DB instance running")
-var addrDB = flag.String("db", "", "DB address to use")
+var (
+	keepDB  = flag.Bool("keep-db", false, "keep test DB instance running")
+	addrDB  = flag.String("db", "", "DB address to use")
+	reuseDB = flag.Bool("reuse-db", false, "re-use running DB Docker container")
+)
 
 func GetDBInstance(pool *dockertest.Pool) (string, func()) {
+	if *reuseDB {
+		resource, ok := pool.ContainerByName("postgres")
```

Comment:
Perhaps read name from an env var? Makes it a little more flexible

Reply:
Done

Reply:
Sorry for being annoying: why not accept a PG_TEST_URI and use that? Could be a docker container or simply a locally running pg instance, which might aid in local debugging. On a local dev machine, one could set it and forget it..

Reply:
Agree, I can use `-db` the above code just make it 'auto-discover' as it extracts the connection information from the container.